doc/source/programs/gdal_raster_reproject.rst

```diff
@@ -129,6 +162,111 @@ Standard options
+    warping option (see :cpp:member:`GDALWarpOptions::papszWarpOptions`) to be
+    set to ``YES``, if it is not explicitly set.
+
+    If ``--srcnodata`` is not explicitly set, but the source dataset has nodata values,
+    they will be taken into account, with ``UNIFIED_SRC_NODATA`` at ``PARTIAL``
+    by default.
+
+.. option:: --dstnodata <DSTNODATA>
+
+    Set nodata values
+    for output bands (different values can be supplied for each band).  If more
+    than one value is supplied all values should be quoted to keep them together
+    as a single operating system argument.  New files will be initialized to this
+    value and if possible the nodata value will be recorded in the output
+    file. Use a value of ``None`` to ensure that nodata is not defined.
+    If this argument is not used then nodata values will be copied from the source dataset.
+    Note that a number of output formats, including GeoTIFF, do not support
+    different per-band nodata values, but a single one for all bands.
+
+.. option:: --addalpha
+
+    Create an output alpha band to identify nodata (unset/transparent) pixels.
+    Value 0 is used for fully transparent pixels, and for Byte datasets, 255
+    for fully opaque pixels.
+
+.. option:: --wo, --warp-option <NAME>=<VALUE>
+
+    Set a warp option.  The :cpp:member:`GDALWarpOptions::papszWarpOptions` docs show all options.
+    Multiple options may be listed.
+
+.. option:: -to <NAME>=<VALUE>
+
+    Set a transformer option suitable to pass to :cpp:func:`GDALCreateGenImgProjTransformer2`.
+    See :cpp:func:`GDALCreateRPCTransformerV2()` for RPC specific options.
+
+.. option:: --et, --error-threshold <ERROR-THRESHOLD>
+
+    Error threshold for transformation approximation, expressed as a number of
+    source pixels. Defaults to 0.125 pixels unless the ``RPC_DEM`` transformer
+    option is specified, in which case an exact transformer, i.e.
+    ``err_threshold=0``, will be used.
+
+Nodata / source validity mask handling
+--------------------------------------
+
+Invalid values in source pixels, either identified through a nodata value
+metadata set on the source band, a mask band, an alpha band (for an alpha band,
+a value of 0 means invalid. Other values are used for blending values) or the use of
+:option:`--srcnodata` will not be used in interpolation.
+The details of how it is taken into account depends on the resampling kernel:
+
+- for nearest resampling, for each target pixel, the coordinate of its center
+  is projected back to source coordinates and the source pixel containing that
+  coordinate is identified. If this source pixel is invalid, the target pixel
+  is considered as nodata.
+
+- for bilinear, cubic, cubicspline and lanczos, for each target pixel, the
+  coordinate of its center is projected back to source coordinates and a
+  corresponding source pixel is identified. If this source pixel is invalid, the
+  target pixel is considered as nodata.
```

Comment:
In this case, valid pixels within the kernel radius would not be considered? Maybe clarify this.

Reply:
clarification added